### ISSUE TITLE
Slow Loading Devices and No Response

### ISSUE THREAD

**Issue by Penguinwastaken** [homebridge/homebridge#2351]
### Expected Behavior

When opening the Home app, devices without Homebridge load immediately or within 1-3 seconds. The devices that are connected from the Homebridge should do the same, at least in 5 to 10 seconds (this is already very long time).

### Current Behavior

When opening the Home app, devices without Homebridge load immediately or within 1-3 seconds (as mentioned above). The Homebridge devices load for a very long time and then a "No response" message will appear. As soon as I click on the device it loads immediately and works. However, this is not really a comfortable function.

### Steps to Reproduce (for bugs)

Installed Plugins:
- homebridge-mi-air-purifier v1.2.2
- homebridge-config-ui-x v4.6.3
- homebridge-magichome v0.3.0
- homebridge-tuya-web v0.1.7
- homebridge-wol v3.2.4

All plugins are used as described in the Read Me of those plugins.

Installed Homebridge via Homebridge App from the iOS AppStore on a Raspberry Pi 3B+.

### Versions

- **Homebridge**: `0.4.50`
- **Node**:       `9.90`
- **OS**:         Raspbian GNU/Linux 10 (buster)
- **NPM**:       `5.60`

### Configuration

{
    "accessories": [
        {
            "accessory": "NetworkDevice",
            "name": "Gaming PC",
            "mac": "1`*******",
            "ip": "192.168.1.3--",
            "shutdownCommand": "net rpc shutdown --ipaddress 192.168.1.3-- --user username%password"
        },
        {
            "accessory": "MagicHome",
            "purewhite": false,
            "name": "LED TV",
            "ip": "192.168.1.10-",
            "setup": "RGBW"
        },
        {
            "accessory": "MiAirPurifier",
            "name": "Mi Air Purifier",
            "ip": "192.168.1.10-",
            "token": "----------",
            "showTemperature": true,
            "showHumidity": true,
            "showAirQuality": true,
            "showLED": true,
            "showBuzzer": false
        }
    ],
    "bridge": {
        "name": "HomebridgeOne",
        "username": "----------",
        "port": 51826,
        "pin": "----------"
    },
    "platforms": [
        {
            "platform": "TuyaWebPlatform",
            "name": "TuyaWebPlatform",
            "options": {
                "username": "----------@icloud.com",
                "password": "----------",
                "countryCode": "49",
                "platform": "smart_life",
                "pollingInterval": 5
            }
        },
        {
            "name": "Config",
            "port": 8080,
            "auth": "form",
            "theme": "dark-mode",
            "tempUnits": "c",
            "sudo": true,
            "log": {
                "method": "file",
                "path": "/var/log/homebridge.log",
                "command": "/var/log/homebridge.stdout.log"
            },
            "accessoryControl": {
                "debug": true
            },
            "platform": "config"
        }
    ]
}

```json

```

### Homebridge Log / Command Output

```

```

**Comment by ukxxx** [comment]
Have exactly same issue. 
Plugins:
homebridge-away-mode v1.1.0
homebridge-broadlink-rm v3.5.5
homebridge-config-ui-x v4.6.3
homebridge-magichome v0.3.0 homebridge-ewelink-max v0.1.17
homebridge-mi-aqara v0.8.0

{
    "bridge": {
        "name": "Homebridge",
        "username": "***",
        "port": 51826,
        "pin": "***"
    },
    "description": "***",
    "accessories": [
        {
            "accessory": "AwayMode",
            "name": "Away Mode",
            "sensorNames": [
                "Trigger 1",
                "Trigger 2"
            ],
            "minOffTime": 300,
            "maxOffTime": 1800,
            "minOnTime": 1800,
            "maxOnTime": 3600
        },
        {
            "accessory": "MagicHome",
            "name": "Color Curtain",
            "ip": "192.168.1.100",
            "setup": "RGBW",
            "purewhite": false
        }
    ],
    "platforms": [
        {
            "platform": "MiAqaraPlatform",
            "name": "Mi Aqara",
            "gateways": {
                "7c49eb17994f": "***"
            },
            "defaultValue": {
                "7c49eb17994f": {
                    "Gateway_LightSensor": {
                        "name": "Освещенность"
                    },
                    "Gateway_Switch_JoinPermission": {
                        "name": "Разрешение на подключение",
                        "disable": false
                    },
                    "WaterDetector_LeakSensor1": {
                        "name": "LeakBathroom"
                    },
                    "WaterDetector_LeakSensor2": {
                        "name": "LeakKitchen"
                    },
                    "Gateway_Lightbulb": {
                        "name": "Свет шлюза",
                        "serviceType": "Lightbulb"
                    }
                }
            }
        },
        {
            "name": "Config",
            "port": 8080,
            "auth": "form",
            "theme": "dark-mode",
            "tempUnits": "c",
            "sudo": true,
            "log": {
                "method": "systemd",
                "service": "homebridge"
            },
            "platform": "config"
        },
        {
            "platform": "eWeLink",
            "name": "eWeLink",
            "email": "***",
            "password": "***",
            "imei": "343A30C3-CF4E-4568-A2E6-9433C248EB55"
        },
        {
            "platform": "BroadlinkRM",
            "name": "Broadlink RM",
            "hideScanFrequencyButton": true,
            "hideLearnButton": false,
            "accessories": [
                {
                    "name": "TV",
                    "type": "switch",
                    "disableAutomaticOff": true,
                    "data": {
                        "on": "2600800000012c92151114111510151114111536141115101536153614361536153615101536153515111535151115101536151015361510153615101536153515111535151115351500051500012c4815000c6000012c4815000c6000012c4815000c6000012c4815000c6000012c4815000c6000012c4716000c6000012b4815000d050000000000000000",
                        "off": "2600800000012c92151114111510151114111536141115101536153614361536153615101536153515111535151115101536151015361510153615101536153515111535151115351500051500012c4815000c6000012c4815000c6000012c4815000c6000012c4815000c6000012c4815000c6000012c4716000c6000012b4815000d050000000000000000"
                    },
                    "onSendCount": 2,
                    "offSendCount": 1,
                    "interval": 0.5
                },
                {
                    "name": "Conditioner",
                    "type": "air-conditioner",
                    "maxTemperature": 28,
                    "minTemperature": 10,
                    "pseudoDeviceTemperature": 24,
                    "defaultCoolTemperature": 21,
                    "defaultHeatTemperature": 26,
                    "ignoreTemperatureWhenOff": true,
                    "sendTemperatureOnlyWhenOff": false,
                    "turnOnWhenOff": false,
                    "data": {
                        "on": "2600ca00939014341410143414341410141014341410141014341411141014341434141014341410143414101434143414341434143414341411133514101410141014101410143414341410141113111410141014101410141014341434143414341434143414ab929014341410143413351410131212351312121213351311131113351335131113351311123613121137123612351335133512371212113711131212121212121212123612361212121311131113111312121212121212361236133513351335133514000d050000000000000000000000000000",
                        "off": "2600ca00919112361212123612361213111311371113121212361212121212361236121212371113123612361236123611131236123612361212121212121213113612131113123612361236121212121212121212131113111312121236123612361236123612ad919112361212123612361212121212361212121311371113111312361236121212361212123612361236123612121236123612361213111311131113123612121212123612361434141014101411131113111311141014101434143414341434143412000d050000000000000000000000000000",
                        "temperature17": {
                            "pseudo-mode": "cool",
                            "data": "2600ca00938f14341410143414341410141014341410141014341411141014341434141014341410143414101434143414341434143414341410143414111410141014101410141014111311141014101410141014101434143414341434143414341434143414ab938f14341410143414341411131114341410141014341410141014341434141014341411143414101434143414341434143414341410143414101411131113111410141014101410141014101411131114101434143414341434143414341434143413000d050000000000000000000000000000"
                        },
                        "temperature18": {
                            "pseudo-mode": "cool",
                            "data": "2600ca00948f14341410143414341410141014341411131113351311141014341434141014341410143414101434143414341434143414341411133514101410141014101410141113111410143414101410141014101434143414341411133514341434143413ab948e14341410143513341411141014341410141014341410141014341434141113351410143414101434143414341434143414341410143414101411131114101410141014101410143414111410141014101434143414341410143414341434143414000d050000000000000000000000000000"
                        },
                        "temperature19": {
                            "pseudo-mode": "cool",
                            "data": "2600ca00938f14341410143414341410141014341410141114341410141014341434141014341410143513101434143414341434143414351311143414101410141014101410141113111434143413111410141014101434143414101410143414341434143414ab938f14341410143414341411131114341311141014341410141014341434141014341411143414101434143414341434143414341410143414101410141113111410141014101434143414101410141014111335143414101410143414341434143413000d050000000000000000000000000000"
                        },
                        "temperature20": {
                            "pseudo-mode": "cool",
                            "data": "2600ca00948e14341410143513341411141014311710141014341410141014341434141014351311143414101434143414341434143414341410143414101410141113111410141014101434141014101410141113111434143414101434143414341434143413ac938f14341410143414341410141014341410141014351311141014341434141014341410143414101434143414341434143414341411143414101410141014101410141113111434141014101410141014101434143414101434143414341434143414000d050000000000000000000000000000"
                        },
                        "temperature21": {
                            "pseudo-mode": "cool",
                            "data": "2600ca00948f14341410143414341410141014341410141114341410141014341434141014341410143414101434143414341434143414341411143414101410141014101410141113351434141014101410141014101434141014111334143414341434143414ab948e14341410143414341411131114341410141014341410141014341434141014341411143414101434143414341434143414341410143216101410141014111311141014341434141014101410141014111434141014101434143414341434143413000d050000000000000000000000000000"
                        },
                        "temperature22": {
                            "pseudo-mode": "cool",
                            "data": "2600ca00938f14341410143414341410141114341311141014341410141014341434141014341411133414111434143414341434143414341410143414101410141014101411141014341434143414101410141014101434141113111410143414341434143413ab948e14341410143513341411141014341410141014341410141014341434141014341411143414101434143414341434143414341410143414101410141113111410141014341434143414101410141014111335141014101410143414341434143414000d050000000000000000000000000000"
                        },
                        "temperature23": {
                            "pseudo-mode": "cool",
                            "data": "2600ca00938f14341410143414341410141113341411141014341410141014341434141014341411143414101434143414341434143414341410143414101410141113111410141014341410143414101411131114101434141014341410143414341434143414ab938f14341410143414341410141014341410141014341411131114341335131114341410143414101434143414341434143414341410143414111311141014101410141014341410143513111410141014101434141014341411133514331534133513000d050000000000000000000000000000"
                        },
                        "temperature24": {
                            "pseudo-mode": "cool",
                            "data": "2600ca00929013351311143414341410141014341410141014341410141113341434141114341410143414101434143414341434143414341410143216111311131114101410141014341410141113111410141014101434141014341434143414341434143414ab938f14341410143414341410141014341410141113351311141014341434141014341410143414101434143414341434143414341411133513111410141014101410141014341411131114101410141014101434141113341434143414341434143414000d050000000000000000000000000000"
                        },
                        "temperature25": {
                            "pseudo-mode": "cool",
                            "data": "2600ca00939014341410143414341410141014341410141014341411131114341434141014341410143414101434143414341433153414341410143414111311141014101410143414341410141014101411131114101410141014341434143414341434143414ab938f14341410143414341410141014341410141113351410141014341434141014341410143414101434143414341434143414341411143414101410141014101410143414341410141113111410141014101410141014341533143414341434143414000d050000000000000000000000000000"
                        },
                        "temperature26": {
                            "psudo-mode": "heat",
                            "data": "2600ca00938d16341410143414341410141113341411141014341410141014341434141014341411143413111434143414341434143414341410143414101410141014111311143414341410143414341434141014101410141014341411141014101434143413ac938f14341410143414341410141014341410141014341411131114341434141014341410143414101434143414341434143414341410143414111311141014101410143414341410143414341434141014111311141014341410141014101434143414000d050000000000000000000000000000"
                        },
                        "temperature27": {
                            "pseudo-mode": "heat",
                            "data": "2600ca00938f14341410143414341410141014341410141114341410141014341434141014341410143414101434143414341434143315351410143414101410141014101411133513111410143414341434141014101410143414341410141014111335143413ac938f13341411143414341410141014341410141014341410141113341434141113351410143414101434143414341434143414341410143414101411131114101410143414101410143414341434141014101411143413341411141014101434143414000d050000000000000000000000000000"
                        },
                        "temperature28": {
                            "pseudo-mode": "heat",
                            "data": "2600ca00929014341410143414341410141014341410141114341410141014341434141014341410143414111335133414341434143414351410143414101410141014101411133513111410141014341434141014101411133414341434141113111434143413ac938f14341410143414341410141014341410141014341411141014341434141014341410143414101434143414341434143414341411133514101410141014101410143414111311141014341433151014101410143414341434141014101434143414000d050000000000000000000000000000"
                        }
                    }
                }
            ]
        }
    ]
}

**Comment by csupi003** [comment]
For me the problem was homebridge-eWeLink-max. There’s already a new plugin [Homebridge-EWeLink-krasnov](https://github.com/VictorKrasnov/homebridge-ewelink#readme) fixing the 503 error.

**Comment by ukxxx** [comment]
> For me the problem was homebridge-eWeLink-max. There’s already a new plugin [Homebridge-EWeLink-krasnov](https://github.com/VictorKrasnov/homebridge-ewelink#readme) fixing the 503 error.

Just try to use plugin you recommend. Unfortunately, still have problem.

**Comment by a-x-** [comment]
Guys and girls, use details html tag for hiding long details under cut, please

**Comment by a-x-** [comment]
+1 to problem

I have single plugin, [**GPIODevice** accessory](https://github.com/dubocr/homebridge-gpio-device).

So, I'm going to file the same issue to GPIODevice

btw, Do you know any debug methods for homebridge and its ecosystem?

**Comment by NorthernMan54** [comment]
DEBUG=* homebridge

Gives detailed information about the messages between the Home app and homebridge

**Comment by DJBenson** [comment]
I have pretty much had this issue since I started using Homebridge a couple of years ago. One way I’ve mitigated it is by putting all my “critical” devices (which tend to be more stable, such as my Honeywell Evohome heating, Philips Hue lights) one one instance of Homebridge and everything else on another so at least if one “bad” plugin takes the system down, there is still access to the devices which we need daily.

I’m now in the position where I’m probably going to install a third instance, one for “critical”, one for my daily use devices which are known to be fairly stable and one experimental instance where I can try out new stuff without breaking the whole install.

On the issue itself, I find that if one device doesn’t respond in a reasonable amount of time (no idea what that is though) the rest of the devices on that instance all show as “No response” quite often (not always though). Closing and opening the Home app often resolves the issue but it certainly dents the WAF (Wife Acceptance Factor) which is important with these things 😉

**Comment by ukxxx** [comment]
Last update do not solve the problem. What about you, guys?

**Comment by W0GER** [comment]
I upgraded from Homebridge 0.4.46 to 0.4.53 (currently the latest release). Prior to the upgrade I didn't have any issues in the Apple Home app when viewing accessories; after the upgrade the Apple Home app shows "Not responding" very frequently for devices connected via homebridge. 

I also updated my plugins, however, this didn't seem to make any difference in the "Not responding" issue.  

I have a DEBUG=* log, but it shows some NEST HTTPS calls with auth tokens, I'd prefer not to post here but willing to get it to an admin.
